src/components/SearchInput/index.tsx

```diff
@@ -0,0 +1,82 @@
+import React, { ChangeEvent, FormEvent, Dispatch, SetStateAction } from 'react';
+import { Box, IconButton, InputBase } from '@mui/material';
+import SearchIcon from '@mui/icons-material/Search';
+import { makeStyles } from '@mui/styles';
+import styled from '@emotion/styled';
+
+const SearchContainer = styled(Box)`
+  display: flex;
+  justify-content: center;
+  margin-top: 104px;
+`;
+
+const SearchStyles = styled.form`
+  border-radius: 4px;
+  border: 1px solid #d7d7d7;
+  width: calc(40% - 18px);
+  display: flex;
+  align-items: center;
+`;
+
+const CustomIconButton = styled(IconButton)`
+  &:hover {
+    background: none;
+  }
+`;
+
+type SearchEventType = FormEvent<HTMLFormElement> | React.MouseEvent<HTMLButtonElement, MouseEvent>;
+
+interface PropsType {
+  setSearchValue: Dispatch<SetStateAction<string>>;
+}
+
+const useStyles: any = makeStyles(() => ({
+  inputRoot: {
+    color: 'inherit',
+    width: '100%'
+  },
+  inputInput: {
+    padding: '8px 40px 8px 0'
+  }
+}));
+
+const SeachInput = ({ setSearchValue }: PropsType) => {
+  let search = '';
```

Comment:
Here on line 44, would putting search outside of the component better? If placed here it'd be created new every time the component re-render?